etcd/shuffle.go

```diff
@@ -0,0 +1,19 @@
+package etcd
+
+import (
+	"math/rand"
+)
+
+func shuffleStringSlice(cards []string) {
```

Comment:
could we still use helper function? it could be cleaner if it is `func shuffleStringSlice(cards []string) []string`, and you can copy the slice inside this function.